app/layer-groups/subway.js

```diff
@@ -5,8 +5,8 @@ export default {
   meta: {
     description: 'NYC Subway Lines and Stops - Originally Sourced from NYC DoITT GIS, combined with SI Railway data from Baruch College NYC Mass Transit Spatial Layers | Subway entrances from NYC Open Data',
     url: [
-      'https://planninglabs.carto.com/api/v2/sql?q=SELECT * FROM support_trans_mta_subway_stops',
-      'https://planninglabs.carto.com/api/v2/sql?q=SELECT * FROM support_trans_mta_subway_routes',
+      'https://planninglabs.carto.com/api/v2/sql?q=SELECT * FROM support_trans_mta_subway_stops&format=SHP',
+      'https://planninglabs.carto.com/api/v2/sql?q=SELECT * FROM support_trans_mta_subway_routes&format=SHP',
```

Comment:
@chriswhong curious, what's this for? Separate issue?

Reply:
Yes, it's the only other meta with a carto API call, and it should be a SHP download, not a CSV.